websites/T/Tidal/presence.ts

```diff
@@ -24,6 +24,18 @@ const getLanguages = async () => {
       await presence.getSetting("language")
     );
   },
+  SongQuality = (quality: string): string => {
+    switch (quality) {
+      case "LOSSLESS":
+        return "Hi-Fi";
+      case "HIGH":
+        return "High";
+      case "LOW":
+        return "Normal";
+      default:
+        return "Unknown";
```

Comment:
maybe show the default quality of the website instead of unknown? (Probs normal?)